src/legacy/core_plugins/apm_oss/index.js

```diff
@@ -49,8 +49,7 @@ export default function apmOss(kibana) {
         'transactionIndices',
         'spanIndices',
         'metricsIndices',
-        'onboardingIndices',
-        'cmIndex'
+        'onboardingIndices'
```

Comment:
removed cmIndex from list of indices that are exposed to kibana as following the apm index pattern. This was causing failure in smoke tests in the backport CI. Those test failed because the upgrade assistant would list this index as one which requires an upgrade/reindex to be ready for 8.0. This fix removes the association of this new index with the standard apm index pattern.